src/interface/src/app/app.module.ts

```diff
@@ -38,6 +38,7 @@ import { SignupComponent } from './signup/signup.component';
 import { StringifyMapConfigPipe } from './stringify-map-config.pipe';
 import { TopBarComponent } from './top-bar/top-bar.component';
 import { PlanMapComponent } from './plan/plan-map/plan-map.component';
+import { ProgressPanelComponent } from './plan/progress-panel/progress-panel.component';
```

Comment:
thought for future cleanup: should we create a PlanModule and include all the subcomponents there instead of in AppModule?

Reply:
Yes! :D